pyaqueduct/client/client.py

```diff
@@ -164,22 +164,23 @@ def get_experiments(
         offset: int,
         title: Optional[str] = None,
         tags: Optional[List[str]] = None,
-        start_date: Optional[date] = None,
-        end_date: Optional[date] = None,
+        start_datetime: Optional[datetime] = None,
+        end_datetime: Optional[datetime] = None,
     ) -> ExperimentsInfo:
         """
         Get a list of experiments
 
         Args:
-        - limit (int): Number of experiments to be fetched
-        - offset (offset): Number of experiments to skip
-        - title (str): Perform search on experiments through title
-        - tags (List[str]): Get experiments having these tags
-        - start_date (date): Start date to filter experiments
-        - end_date (date): End date to filter experiments to
+        - limit: Pagination field, number of experiments to be fetched.
+        - offset: Pagination field, number of experiments to skip.
+        - title: Perform search on experiments through their title and alias.
```

Comment:
I guess, later sometime, we should rename the field of title to something else, maybe `search_str`, `search`, etc.

Reply:
Yes, we have made this decision of using `id` and `eid` for the experiment. That will be a backward incompatible change that happens in a single PR. @safeamiiir has created on aqueductcore repo: https://github.com/AqueductHub/aqueductcore/issues/82